### ISSUE TITLE
links under the "category" are only linked to the main page. 

### ISSUE THREAD

**Issue by czhci** [NEUBIAS/bise#56]
we should add their corresponding pages: training material (XX), software (XX), etc.

**Comment by PerrineGilloteaux** [comment]
It was a bug in the link (no/ should have been set).

**Comment by PerrineGilloteaux** [comment]
commit e786f46c0f67bedf0c2f29f3c8e621420bfc2191

**Comment by miura** [comment]
Now deployed, but is this OK that clicking any of the category goes always to http://biii.eu/all-content ? Not a list of pages for a specific category? @PerrineGilloteaux @czhci

**Comment by PerrineGilloteaux** [comment]
For now it was a quick fix where people can choose to show only one category

**Comment by czhci** [comment]
maybe we should try to fix it properly?

**Comment by miura** [comment]
This is now implemented as it should be. (@czhci  @PerrineGilloteaux ). I will push this later, but just to notify you that you do not need to work on this. 
![image](https://user-images.githubusercontent.com/272781/85439051-de6dad00-b58c-11ea-95f6-bcefc7d35300.png)

In association with this, custom page names such as "/software-view", "/training-view" were changed to "/software" and "training-material", as this was the problem linking these views. In addition, I created "/dataset" for the same purpose. Maybe these pages need to be polished.
